src/mrnet/core/rates.py

```diff
@@ -41,47 +41,39 @@ class ReactionRateCalculator(MSONable):
 
     def __init__(self, reactants, products, transition_state):
 
-        self.reactants = reactants
-        self.products = products
         self.transition_state = transition_state
 
         # Assume rate law is first-order in terms of each reactant/product
-        rate_law = {
-            "reactants": {ii: 1 for ii in range(len(self.reactants))},
-            "products": {jj: 1 for jj in range(len(self.products))},
+        self.rate_law = {
+            "reactants": np.ones(len(reactants)),
+            "products": np.ones(len(products)),
         }
 
-        self.rate_law = rate_law
+        # Store relevant information from reactants, products, rather than storing obj
+        # Here, they are stored as np arrays, which have a smaller footprint
+        self.pro_energy = sum([p.energy for p in products])
+        self.rct_energy = sum([r.energy for r in reactants])
 
-    @property
-    def net_energy(self):
-        """
-        Calculate net reaction energy (in eV).
-        """
-        rct_energies = [r.energy for r in self.reactants]
-        pro_energies = [p.energy for p in self.products]
+        self.pro_enthalpy = sum([p.enthalpy for p in products])
+        self.rct_enthalpy = sum([r.enthalpy for r in reactants])
```

Comment:
MoleculeEntries, strictly speaking, do not HAVE to have enthalpy and entropy. This is fine for now, but eventually, we'll want to make sure we fail gracefully in that case.